templates/base.html

```diff
@@ -1,140 +1,140 @@
-			}
-			a:hover {
-				color: #4078C0;
-				text-decoration: underline;
-			}
-			a:active {
-				color: #4078C0;
-				text-decoration: underline;
-			}
-		</style>
+        <style type="text/css">
+            body {
+                margin: 0px;
+                background: #fff;
+            }
+            #wrapper {
+                min-height:100%;
+                position:relative;
+            }
+            #header {
+                position: fixed;
+                height: 50px;
+                width: 100%;
+                top: 0px;
+                background: #2980b9;
+                border-bottom:1px solid #dedede;
+                margin: 0px auto;
+                padding-left: 10px;
+                font-family: 'Lato', sans-serif;
+                font-weight: 900;
+                display: table;
+                z-index: 10000;
+            }
+            #content {
+                margin: 60px 1em 1em 1em;
+                font-family: 'Lato', sans-serif;
+                font-weight: 400;
+                padding-bottom: 20px;
+            }
+            #footer {
+                text-align: center;
+                vertical-align: bottom;
+                height: 20px;
+                bottom: 0;
+                font-family: Arial, Helvetica, sans-serif;
+                font-size: 0.8em;
+            }
+            .title {
+                text-shadow: 1px 2px 2px #2980b9, 0 0 0 #000, 1px 2px 2px #2980b9;
+                color: rgba(255, 255, 255, 0.8);
+                font-size: 30px;
+                display: table-cell;
+                vertical-align: middle;
+                white-space: nowrap;
+                width: 1px;
+            }
+            a.title {
+                color: inherit;
+            }
+            .headline {
+                color: #eee;
+                font-size: 20px;
+                display: table-cell;
+                vertical-align: middle;
+                text-align: center;
+            }
+            .alt tr:nth-child(even) {
+                background-color: #F4F4F4;
+            }
+            .alt tr:nth-child(odd) {
+                background-color: #FAFAFA;
+            }
+            a:link {
+                color: #4078C0;
+                text-decoration: none;
+            }
+            a:visited {
+                color: #4078C0;
+                text-decoration: none;
+            }
+            a:hover {
+                color: #4078C0;
+                text-decoration: underline;
+            }
+            a:active {
+                color: #4078C0;
+                text-decoration: underline;
+            }
+        </style>
 {{template "head" .}}
-	</head>
-	<body>
-		<div id="wrapper">
-			<div id="header">
-				<div class="title">
-					<a href="http://github.com/etix/mirrorbits" target="_blank" style="text-decoration: none; color: inherit;">Mirrorbits <i class="fa fa-globe" style="vertical-align: middle;" aria-hidden="true"></i></a>
```

Comment:
Please keep `https` here.

Reply:
currently your master branch is set to http @ <https://github.com/etix/mirrorbits/blob/master/templates/base.html#L127/>

this makes it https @ https://github.com/etix/mirrorbits/pull/119/files#diff-690b81fad8df2a1f1ce37c846641d9247e3472d244f17039d2910a5ed0c98d5eR127

Reply:
plz review again @lfauvet-bc

Reply:
plz review again @lfauvet-bc  @etix